_positions/front-end-designer.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Front End Designer
```

Comment:
@shawnbot and I talked a bit about this. We liked either Web Designer or Front End Designer, but in the end figured that as long as the description told the story of what we want, and the title had the word 'designer' in it so we got people with design as well as code skills, we were happy.